experimental/framework/tree-react-api/api-report/tree-react-api.alpha.api.md

```diff
@@ -5,15 +5,10 @@
 ```ts
 
 // @public
-export interface IReactTreeDataObject<TSchema extends ImplicitFieldSchema> extends ITreeDataObject<TSchema> {
```

Comment:
Note that this is an `@fluid-experimental` library, so it is fine for us to make breaking changes here.

Reply:
This is a published package though, so it would be good provide a changeset detailing the impact.

Looks like ITreeDataObject is removed with no external facing replacement, generally the apis in this package changed, and all existing documents created using them will be incompatible?